tests/test_extract_inventory_numbers.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import re
+from extract_inventory_numbers import _compile_regex
+
+
+class TestRegEx(unittest.TestCase):
+
+    def test_regex(self):
+        # test cases comprised of tuples
+        # where the first item is the input string
+        # and the second item is the expected output derived from FTVA specs
+        test_cases = (
+            ("HFA27M_Reel", "HFA27M"),
+            ("VA13161T_KTLA", "VA13161T"),
+            ("M190816Medea2", "M190816"),
+            ("XFE1915MX", "XFE1915"),  # invalid suffix case
+            ("Randy_Requiem1", ""),
+            (
+                "XFE4098M_XFF104M_DamagedLives_Finals",
+                "XFE4098M|XFF104M"
+            ),  # multi-match to pipe-delimited string case
+            (
+                "XVE779T_ZVE780T_OneNightStand_WorldOfLennyBruce_CaptureFiles_SD_2997FPS_YUV",
+                "XVE779T"
+            )  # close-but-no-cigar case (ZVE780T does not have valid prefix)
```

Comment:
Please tweak this comment to make it clearer what's being tested: 1 good prefix, 1 bad.  I had to read it twice to be sure it wasn't a `XVE` vs `ZVE` typo.